examples/meta/generator/targets/python.json

```diff
@@ -61,7 +61,8 @@
             "get_int_vector": "$object.get($arguments)",
             "get_real": "$object.get($arguments)",
             "get_real_vector": "$object.get($arguments)",
-            "get_real_matrix": "$object.get($arguments)"
+            "get_real_matrix": "$object.get($arguments)",
+            "put_machine": "$object.put($arguments)"
```

Comment:
this will not work as we cannot extract the arguments yet and you would need to pass the second argument to the `machine` factory

Reply:
we can do this once #4490 is solved